src/validators/stop_times.rs

```diff
@@ -30,11 +38,40 @@ pub fn validate(gtfs: &gtfs_structures::Gtfs) -> Vec<Issue> {
         })
     });
 
-    wrong_stops.into_iter().map(|(_, v)| v).collect()
+    wrong_stops.into_iter().map(|(_, v)| v)
```

Comment:
a bit unrelated, but can’t we use https://doc.rust-lang.org/std/collections/struct.HashMap.html#method.into_values ?

Reply:
yep, done